app-emulation/virt-manager/metadata.xml

```diff
@@ -1,27 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<name>Filip Kobierski</name>
+		<email>fkobi@pm.me</email>
+	</maintainer>
 	<maintainer type="project">
 		<email>virtualization@gentoo.org</email>
 		<name>Gentoo Virtualization Project</name>
 	</maintainer>
 	<use>
-		<flag name="policykit">
```

Comment:
I don't see how removing the description is an improvement. You also don't know about people who _did_ read it and therefore had it solve their problem.